### ISSUE TITLE
‘uint16_tword’ undeclared

### ISSUE THREAD

**Issue by grassbl8d** [qmk/qmk_firmware#8281]
Hi,

I'm getting this error when copmiling lately.

quantum/quantum.c:646:5: error: ‘uint16_tword’ undeclared (first use in this function)
     uint16_tword = (number >> 16);
     ^
quantum/quantum.c:646:5: note: each undeclared identifier is reported only once for each function it appears in
quantum/quantum.c:647:15: error: ‘word’ undeclared (first use in this function)
     send_word(word);

I think this might be related to a recent change . Can't compile firmware right now am getting this error.

![image](https://user-images.githubusercontent.com/93050/75628298-804e4000-5c12-11ea-8e87-1bd92e546ba6.png)

Thanks

**Comment by maciejmatczak** [comment]
https://github.com/qmk/qmk_firmware/commit/1ec8a7205f1719496e043776edcc972125fc57e3

It seems that formatting without full CI let that issue through...

**Comment by fauxpark** [comment]
Duplicate of https://github.com/qmk/qmk_firmware/pull/8280
